cub/cub/block/block_scan.cuh

```diff
@@ -412,12 +412,11 @@ public:
   //! Each thread contributes one input element.  Instead of using 0 as the block-wide prefix, the call-back functor
   //! ``block_prefix_callback_op`` is invoked by the first warp in the block, and the value returned by
   //! *lane*\ :sub:`0` in that warp is used as the "seed" value that logically prefixes the thread block's
-  //! scan inputs. Also provides every thread with the block-wide ``block_aggregate`` of all inputs.
```

Comment:
Just to make sure I'm not misunderstanding: To my understanding, the `block_aggregate` _is_ returned for every thread. It's just that the `block_aggregate` _does not incorporate the `exclusive_prefix`_, returned from `block_prefix_callback_op`?

Reply:
Ah, got your point. This is the documentation of an overload that does not take a `block_aggregate` parameter at all!